examples/LexActivator.pas

```diff
@@ -175,6 +175,110 @@ procedure SetCustomDeviceFingerprint(const Fingerprint: UnicodeString);
 
 procedure SetLicenseKey(const LicenseKey: UnicodeString);
 
+(*
+    PROCEDURE: SetReleaseVersion()
+
+    PURPOSE: Sets the current release version of your application.
+    The release version appears along with the activation details in dashboard.
+
+    PARAMETERS:
+    * ReleaseVersion - string in following allowed formats: x.x, x.x.x, x.x.x.x.
+
+    EXCEPTIONS: ELAProductIdException, ELAReleaseVersionFormatException
+*)
+
+procedure SetReleaseVersion(const ReleaseVersion: UnicodeString);
+
+(*
+    PROCEDURE: SetReleasePlatform()
+
+    PURPOSE: Sets the release platform e.g. windows, macos, linux
+    The release platform appears along with the activation details in dashboard.
+
+    PARAMETERS:
+    * ReleasePlatform - release platform e.g. windows, macos, linux.
+
+    EXCEPTIONS: ELAProductIdException, ELAReleaseVersionFormatException, ELAReleasePlatformLengthException
+*)
+
+procedure SetReleasePlatform(const ReleasePlatform: UnicodeString);
+
+(*
+    PROCEDURE: SetReleaseChannel()
+
+    PURPOSE: Sets the release channel e.g. stable, beta.
+    The release channel appears along with the activation details in dashboard.
+
+    PARAMETERS:
+    * ReleaseChannel - release channel e.g. stable
+
+    EXCEPTIONS: ELAProductIdException, ELAReleaseChannelLengthException
+*)
+
+procedure SetReleaseChannel(const ReleaseChannel: UnicodeString);
+
+(*
+    PROCEDURE: SetReleasePublishedDate()
+
+    PURPOSE: Sets the release published date of your application.
+
+    PARAMETERS:
+    * ReleasePublishedDate - unix timestamp of release published date.
+
+    EXCEPTIONS: ELAProductIdException
+*)
+
+procedure SetReleasePublishedDate(ReleasePublishedDate: LongWord);
+
```

Comment:
@muneebkq We have used `TDateTime` wherever the timestamp is returned. Please make sure it remains consistent.